android/src/main/java/com/instabug/instabugflutter/InstabugFlutterPlugin.java

```diff
@@ -114,7 +117,7 @@ public void start(Application application, String token, ArrayList<String> invoc
             String key = invocationEvents.get(i);
             invocationEventsArray[i] = ArgsRegistry.getDeserializedValue(key, InstabugInvocationEvent.class);
         }
-        new Instabug.Builder(application, token).setInvocationEvents(invocationEventsArray).build();
+        new Instabug.Builder(application, token).setInvocationEvents(invocationEventsArray).setReproStepsState(State.DISABLED).build();
```

Comment:
This should be enabled by default.

Reply:
@HeshamMegid  We need to disable the native repro steps for android to use the new private APIs